### PR TITLE
Add contributing guideline to the repo.

### PR DESCRIPTION
**Pull request by Juke34**
Add contributing guideline to the repo

### CODE REVIEW

**Comment by ropolomx** [comment]
Addresses #11.

**Review by greysaruman — COMMENTED** [review]
Thank you Jacques, this guidelines document looks great! I think it would be perfect if we also include an example of a well-described issue in the "Did you find a bug?" section.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Contributing guidelines
+
+We thank you in advance :thumbsup: :tada: for taking the time to contribute, whether with *code* or with *ideas*, to the project.
+
+
+## Did you find a bug?
```
Comment (ropolomx) [inline CONTRIBUTING.md:6]: As @greysaruman  suggested, it would be great to have an example of a real bug in this section.

**Review by ropolomx — COMMENTED** [review]
The contributing guidelines look great! The only comment would be about adding an example to the "Did you find a bug?" section. This is excellent work and very helpful. It will be a work in progress, especially writing language accessible to non-technical users. I suggest that we add this now, and then we can refine or polish it more.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Juke34) [reply CONTRIBUTING.md:6]: I added an issue template in #16.
I keept a standard one from github, feel free to suggest any update.

**Comment by Juke34** [comment]
This PR does not touch the code or the displayed content. So I go ahead with merging... it can be modified/improved at any time

**Comment by ropolomx** [comment]
Thank you @Juke34 !!

**Pull request merged**